kevlar/call.py

```diff
@@ -83,27 +83,9 @@ def dedup(callstream):
 
 
 def prelim_call(targetlist, querylist, match=1, mismatch=2, gapopen=5,
-                gapextend=0, ksize=31, refrfile=None, mindist=5,
+                gapextend=0, ksize=31, refrfile=None, debug=False, mindist=5,
                 logstream=sys.stderr):
-    """Implement the `kevlar call` procedure as a generator function.
-
-    Input is the following.
-    - an iterable containing one or more target sequences from the reference
-      genome, stored as khmer or screed sequence records
-    - an iterable containing one or more contigs assembled by kevlar, stored as
-      khmer or screed sequence records
-    - alignment match score (integer)
-    - alignment mismatch penalty (integer)
-    - alignment gap open penalty (integer)
-    - alignment gap extension penalty (integer)
-    - mates of interesting reads, in case these are needed to distinguish
-      between multiple best hist (filename)
-    - reference file to which mates of interesting reads, if any, will be
-      mapped to disambiguate multi-mapping contigs
-
-    The function yields tuples of target sequence name, query sequence name,
-    and alignment CIGAR string
```

Comment:
Great example of how documentation can easily get out of sync with the code it's describing.